CHANGELOG

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Added types for HTTPStatusCodes and HTTPResponse (SCC-4172)
```

Comment:
Add a new "Prerelease" or "Unreleased" title. This update should go under there instead of the `1.2.0` release.